bindata/deployment/metallb.yaml

```diff
@@ -207,6 +207,8 @@ spec:
         kubernetes.io/os: linux
       securityContext:
         runAsNonRoot: true
+        {{ if not .IsOpenShift }}
```

Comment:
Can you add this logic to the generation script (https://github.com/metallb/metallb-operator/blob/main/hack/generate-metallb-manifests.sh)? I think this will get overridden otherwise.

Reply:
ok will look into that

Reply:
shouldn't we use quilt tool here to patch the manifest it would have been much cleaner